dGame/dUtilities/SlashCommandHandler.cpp

```diff
@@ -171,6 +171,19 @@ void SlashCommandHandler::HandleChatCommand(const std::u16string& command, Entit
 	}
 #endif
 
+	if (chatCommand == "togglenameplate" && (Game::config->GetValue("allownameplateoff") == "1" || entity->GetGMLevel() > GAME_MASTER_LEVEL_DEVELOPER)) {
```

Comment:
:smiling_face_with_tear: I'll need to fix this in #996 Note to self also to Emo whow will have to re-review that RM